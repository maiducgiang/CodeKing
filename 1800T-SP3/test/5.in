10
30
472 920
576 12
925 -229
188 -478
161 -215
843 -836
214 764
369 -299
506 -152
746 -573
264 438
283 -939
122 893
207 -594
869 558
955 -121
819 -511
520 -400
374 -695
468 571
100 287
543 -780
852 655
824 -810
253 -626
193 798
233 385
745 675
76 94
765 29
75
740 829
428 776
1 -929
557 354
38 -371
222 -192
901 -134
288 335
42 -314
692 -691
513 -348
663 -525
310 -502
182 -964
187 -422
789 365
251 494
654 -574
112 170
868 73
430 -640
181 -207
624 -663
890 -41
134 -90
702 -661
243 813
748 908
434 -109
75 -368
835 521
710 -266
356 822
394 -517
435 598
728 -675
983 420
737 -321
144 983
788 -699
228 250
805 -480
512 -375
431 -597
569 -548
304 -985
304 -390
901 -58
239 -302
675 677
659 387
358 -421
151 753
579 761
909 131
679 -293
335 670
315 980
4 -230
101 -694
813 28
684 -903
71 310
869 -12
586 387
239 62
981 -890
736 -682
678 271
986 -763
614 916
560 873
723 -959
821 -576
840 -582
65
719 500
431 -925
423 -402
791 -263
18 -758
789 939
849 -17
33 628
401 122
972 -798
630 287
957 773
833 294
134 -596
507 527
655 -307
53 -824
742 323
44 786
465 -724
64 -88
612 273
651 -611
16 789
734 -119
126 -750
122 159
358 -412
906 956
913 -219
98 -448
801 385
26 775
24 606
735 -146
605 -915
887 97
400 -110
568 701
86 -159
370 -3
897 601
194 602
341 -591
739 -241
169 -883
985 446
518 -2
271 -785
240 -939
497 418
989 1
97 768
740 502
526 -529
656 -223
325 -473
170 -823
334 -864
532 -311
941 -339
668 -645
113 629
441 361
599 -321
82
379 555
268 282
73 -593
5 -991
337 -821
308 695
566 761
206 245
444 -118
896 657
545 -252
277 194
311 -527
197 887
425 122
470 902
259 732
131 873
707 -408
636 862
941 -831
157 776
279 -392
175 -525
218 962
898 -995
170 -522
497 -224
757 809
793 278
19 -58
523 -808
710 -296
912 208
890 -105
264 913
707 -791
482 -423
14 324
674 -779
603 -367
996 -674
976 -579
267 -602
829 -264
102 932
232 219
918 26
692 -513
918 -780
915 -78
679 812
121 688
204 424
831 -895
455 906
949 278
80 827
257 235
177 900
519 -580
876 -733
168 -203
633 287
435 902
739 -273
79 358
686 86
998 -836
889 -410
621 -579
624 567
706 -660
204 727
367 786
580 -334
627 281
692 148
65 -76
507 -64
446 -904
710 122
53
860 -100
384 -769
874 458
41 -500
660 -775
637 -721
193 495
236 -506
691 -656
447 -502
175 -492
883 279
263 647
827 -686
852 -843
393 -251
880 -195
232 -468
12 -600
74 -52
981 -60
883 -180
939 763
45 -260
944 -617
927 802
729 689
792 792
726 -735
154 391
385 -513
949 -223
212 871
467 -787
332 -758
208 275
23 -924
293 -498
231 943
435 404
395 -34
807 963
983 -143
849 -521
424 -191
665 -406
640 43
762 -950
194 237
976 -268
57 398
937 968
252 -50
98
295 -526
244 -558
872 42
158 916
689 -174
963 377
819 -464
787 -901
244 775
727 -912
721 318
570 246
203 498
124 -376
188 489
52 -381
848 30
309 172
492 564
682 -843
911 444
299 240
468 78
997 537
948 -677
88 -474
76 -396
571 -363
115 417
495 -669
83 88
502 115
140 7
892 985
718 42
401 -546
280 -321
38 -223
98 229
849 411
744 523
132 192
597 790
633 460
307 -614
349 257
276 236
590 -195
395 409
460 10
264 -400
561 276
649 654
993 -482
548 -209
174 950
546 910
72 -332
112 808
697 -5
464 -694
210 753
672 -343
165 94
190 -868
491 -405
693 -957
391 624
458 -923
978 -767
186 589
443 -452
973 -975
570 30
71 -927
602 -50
117 528
99 840
37 -960
756 -921
896 -3
257 -224
395 -635
731 432
264 -419
353 187
236 -329
461 301
240 260
384 -229
789 -320
417 -944
535 -540
688 697
771 -549
987 766
545 -266
864 -491
92
657 984
565 -904
908 445
756 562
649 632
524 918
387 161
791 884
870 -510
564 759
206 -37
903 -56
952 -347
107 -633
407 186
255 8
500 691
624 500
708 -137
426 777
592 -836
974 660
378 315
965 692
304 168
716 705
624 -729
475 694
690 405
480 65
94 459
11 -984
518 36
375 752
83 327
443 -856
81 -443
630 508
955 443
163 -536
78 -308
393 -991
310 785
65 -203
845 606
515 253
722 740
579 -217
275 588
549 -102
667 864
96 4
881 -488
286 729
747 21
105 -942
653 496
765 537
577 -102
265 -909
700 -368
450 489
103 -581
800 172
821 -828
833 318
99 -536
626 621
146 -539
104 -992
319 638
353 -191
774 346
658 -589
330 -255
262 698
795 -270
819 -515
438 -13
404 43
294 -412
9 -676
321 719
280 -734
76 155
912 -512
630 356
359 124
481 680
172 -475
521 -195
771 -444
42
996 952
694 -436
805 785
533 48
183 -256
146 852
548 -861
759 208
23 -473
10 873
70 976
346 -205
81 561
844 57
350 36
81 548
97 -508
77 966
942 449
96 -166
520 487
242 -316
538 301
284 -828
624 19
870 -315
349 22
215 -34
756 -685
114 239
625 53
421 -995
548 54
779 994
873 -791
833 928
114 -354
682 -981
21 455
854 -159
886 -773
892 24
33
209 260
990 -462
785 -187
601 -270
201 -777
362 -342
284 -683
614 -823
405 -229
202 631
504 142
614 356
567 -459
210 -472
443 652
553 890
495 -635
912 -416
595 -843
419 -626
457 558
499 -71
513 -347
355 421
428 253
279 592
357 -257
52 640
790 -602
577 37
226 -466
457 378
473 114
99
726 -612
891 -501
64 -295
967 288
287 -915
933 -333
21 -497
196 552
799 -487
875 -445
925 -426
172 -423
119 277
724 683
787 267
897 332
826 -646
883 475
920 229
652 -224
128 812
924 -304
735 465
94 -810
741 -453
600 -423
516 5
619 -677
812 762
279 588
468 -926
247 -56
501 942
699 747
706 328
395 -330
910 -942
763 -250
994 383
531 633
232 140
898 -723
537 525
869 215
529 177
104 891
739 361
171 656
516 38
666 -791
307 -59
957 -284
35 331
742 -393
198 -854
55 425
460 -8
174 -240
109 -762
244 400
16 -698
707 862
438 -351
286 -139
108 -470
402 119
540 682
76 576
146 -187
78 -437
552 762
960 467
348 -95
30 -235
203 900
33 -588
583 -698
180 -55
382 -115
279 -648
636 731
380 -521
484 -678
526 114
963 348
450 -584
94 -275
321 -893
471 869
278 -357
837 -755
215 780
622 -354
514 -629
753 -233
119 -507
358 -344
192 103
861 -738
