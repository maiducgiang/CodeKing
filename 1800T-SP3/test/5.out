3
22
14
5
13
12
18
4
3
12
